vendors/tpl-systemes/vendor.yaml

```diff
@@ -0,0 +1,19 @@
+# ID of the vendor to be used as an internal reference for Actility vendors.
+# Define your own ID following this rule: Maximum of 8 characters in lowercase.
+# Be careful: Once you chose a vendor ID, it cannot be changed later
+id: tpl-systemes
+# Official/commercial name of the vendor to be used as a label.
+name: TPL Systèmes
+# Simple description of the vendor. Maximum 500 characters. Special characters are not allowed ('\n', '\t', '\r', ect...).
+description: TPL Systèmes is a leading designer and manufacturer of radiocommunication equipment and solutions.
+
+# LoRa-Alliance Vendor ID used for QR code https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf. It consists of 4 upper-case hexadecimal characters.
+loRaAllianceVendorID:
+# Logo file name if different from <vendorId>.png (optional).
+logo:
```

Comment:
either name of logo with <vendorId>.png or add the name of the logo file (logo.png in this case)